### DISCUSSION TITLE
Create PR request choosing between multiple templates via UI

### DISCUSSION THREAD

**Discussion by evenSmart** [community/community#13768] [Pull Requests]
Hi there,

in case of multiple PR templates, the user needs to select the template they wish, by writing a select query string inside an URL (c.f. https://github.community/t/multiple-pull-request-template/874). It would be really nice to choose from multiple PR templates with an simple UI element, a drop-down list, say, as can be done when picking a template to create a new repository.

This feature would be really helpful for users that do not know that there's more than one template available and how to choose it.

Thank you!

**Comment by kimdwkimdw** [comment]
I really want it too.


https://github.blog/2018-01-25-multiple-issue-and-pull-request-templates/

Currently, instead of putting`?template=` on the end of the URL, you can make links on default template.


pull_request_template.md 
```
[TEMPLATE1](?template=pr_template_1.md)
[TEMPLATE2](?template=pr_template_2.md)
```

then you can click templates in `Preview` of Pull request.

**Reply by kenji-miyake** [reply]
I also use this workaround. I posted the screen record here.
https://github.com/community/community/discussions/4620#discussioncomment-3383383
